formatTest/unit_tests/expected_output/uncurried.re

```diff
@@ -0,0 +1,115 @@
+f(.);
```

Comment:
Is there an example for `foo => (. bar => baz)`?

Reply:
Will add one in the next commit.
The current implementation output `type f = (foo, . bar) => baz;`

Reply:
`(foo) => (. bar) => baz` looks  more consistent?

Reply:
Currently Reason uses `type f = (foo, bar) => baz;` which represents Ocaml's `type f = foo -> bar -> baz`. I agree that `foo => (. bar => baz)` is easier to read, will change.